heartbeat/LVM-activate

```diff
@@ -820,17 +820,14 @@ lvm_start() {
         if systemd_is_running ; then
         	# Create drop-in to deactivate VG before stopping
 		# storage services during shutdown/reboot.
-		after=$(systemctl show resource-agents-deps.target.d \
-			--property=After | cut -d'=' -f2)
-
-		case "$after" in
-			*" blk-availability.service "*)
-				;;
-			*)
-				systemd_drop_in "99-LVM-activate" "After" \
+		systemctl show resource-agents-deps.target \
+			--property=After | cut -d'=' -f2 | \
+			grep -qE "(^|\s)blk-availability.service(\s|$)"
```

Comment:
`\s` may or may not be supported as shorthand for `[[:space:]]`, depending on the version of grep.
Why not use an actual space? Or even use the same shell "case" logic as before, but like this:
`case " $after " in *" ... "*)` (note the extra space around `$after`)

Reply:
The shell logic before only matched if you got 3 or more hits (and blk... wasnt first or last). We use \s (and the combination with beginning/end of line) with grep in other agents, and noone has complained about it.

Reply:
Yeah, I had used the `case` idiom in 79fb4b2d as an attempt at portability -- I don't remember the details, but I think I had found something less than ideal about grep at the time. `-q` is also not as portable as `>/dev/null 2>&1`.

But none of that mattered, since my two typos (`"$after"` instead of `" $after "`, and `target.d` instead of `target`) broke the whole thing anyway :)